AdminApi/src/AdminApi/ClientApp/src/app/components/quotas/identity/identity-details/identity-details.component.ts

```diff
@@ -93,12 +93,29 @@ export class IdentityDetailsComponent {
         this.tierService.getTiers().subscribe({
             next: (tiers) => {
                 this.tiers = tiers.result;
+                if (this.identity.tierId !== "TIR00000000000000001") {
+                    this.tiers = this.tiers.filter((tier) => tier.id !== this.getQfDTierId());
+                }
```

Comment:
What's this if for?

Reply:
Some how i need to filter out Queued for Deletion tier i thought i'll do it by ID, then i see i made a mistake that i should have checked if the both new parameters are false and with that i can filter out queued for deletion out of an array.